lighthouse-viewer/app/src/lighthouse-report-viewer.js

```diff
@@ -175,19 +175,18 @@ class LighthouseReportViewer {
   }
 
   /**
-   * Opens new tab with compatible report viewer version
+   * Opens legacy viewer in current tab, then renders report
```

Comment:
comment is a bit weird to me since everything that happens after navigating to legacy viewer isn't captured here